HHS/Head-Start-TTADP
# [TTAHUB-975] Fix chart and table discrepancy

## Description of change
Unify DB querying for these two components (the goals table and the goal status chart) so they don't continuously fall out of sync

## How to test
View the goals and objectives tab and try a few different filter combinations. The goal statuses and the goals table should show the same number for counts, both total and for individual goal statuses.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-975


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
